### ISSUE TITLE
bug(Mat-input): placeholder (label) text contrast too low when error for a11y

### ISSUE THREAD

**Issue by frangarcj** [angular/components#25099]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

Placeholder text for (the label) is set at #f44336 on an invalid state. This results in text with a contrast ratio of 3.7:1 when the background color is white.

### Reproduction

https://stackblitz.com/edit/angular-2anknf?file=src/app/./input-error-state-matcher-example.html

Steps to reproduce:
1. Focus on email 
2. Get out of email

### Expected Behavior

The placeholder text for error doesn't have the minimum contrast ratio set forth by WCAG to achieve a Level AA accessibility rating of 4.5:1. Therefore the colour must be changed in order to get a proper result.

### Actual Behavior

This results in text with a contrast ratio of 3.7:1 when the background color is white.

### Environment

- Angular: 13.3.6
- CDK/Material: 13.3.6
- Browser(s): All
- Operating System (e.g. Windows, macOS, Ubuntu): All

**Comment by jelbourn** [comment]
@zarend does this issue exist in the MDC-based version?

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
